public/index.php

```diff
@@ -20,7 +20,7 @@
     Request::setTrustedHosts([$trustedHosts]);
 }
 
-$kernel = new Kernel($_SERVER['APP_ENV'], (bool) $_SERVER['APP_DEBUG']);
+$kernel = new Kernel('smoketest', true);
```

Comment:
@MKodde can't we leave it like that? I want to use the debug bar in smoketest mode. What is the point of hardcoding this?